frontend/packages/dev-console/src/components/project-access/hooks.ts

```diff
@@ -0,0 +1,41 @@
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { ClusterRoleModel } from '@console/internal/models';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export type ClusterRoleKind = K8sResourceCommon & {
+  rules: { [key: string]: string[] }[];
+};
+
+export enum accessRoles {
+  admin = 'Admin',
+  edit = 'Edit',
+  view = 'View',
+}
```

Comment:
I think this is more a defaultAccessRoles not instead of an enum, or?

```suggestion
export const defaultAccessRoles = {
  admin: 'Admin',
  edit: 'Edit',
  view: 'View',
};

```